backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/button/internal/BpkButtonImpl.kt

```diff
@@ -74,16 +74,19 @@ internal fun BpkButtonImpl(
                 .requiredHeight(size.minHeight),
             interactionSource = interactionSource,
             colors = ButtonDefaults.buttonColors(
-                backgroundColor = type.backgroundColor(interactionSource),
+                containerColor = type.backgroundColor(interactionSource),
                 contentColor = type.contentColor(interactionSource),
-                disabledBackgroundColor = if (loading) type.loadingBackgroundColor() else type.disabledBackgroundColor(),
+                disabledContainerColor = if (loading) type.loadingBackgroundColor() else type.disabledBackgroundColor(),
                 disabledContentColor = if (loading) type.loadingContentColor() else type.disabledContentColor(),
             ),
             shape = ButtonShape,
             contentPadding = type.contentPadding,
             elevation = null,
             content = {
-                CompositionLocalProvider(LocalTextStyle provides size.textStyle()) {
+                CompositionLocalProvider(
+                    LocalTextStyle provides size.textStyle(),
+                    LocalContentColor provides androidx.compose.material3.LocalContentColor.current,
```

Comment:
For some time, we're going to have 2 TextStyle/ContentColor composition locals in the project – from MD2 and MD3.

To simplify the migration, I replaced all MD2 CompositionLocals with Bpk CompositionLocals. This should have no effect and Bpk Composition Local actually refers to MD2 Composition locals.

Since our Text and Icon are used in a lot of places and actively use CompositionLocals in there, the default values for these components have been set to BpkCompositionLocal (aka MD2 composition locals). This way, the old code remains unbroken.

The new code, however, might be needed some tweaks. For instance, Button from MD3 sets MD3 CompositionLocals. Our BpkText/Icon wont be able to pick these up unless we set MD2 CompostionLocal = MD3 CompositionLocal like we're doing here.

These changes will only be temporary as we perform the migration, and it will only be affecting the MD3 migrated components (those that provide the containers with CompositionLocals). Once we finish the migration, we're going to point BPK CompositionLocals to MD3 and remove all the usages of non-BPK composition locals.